data/subset/expression-matrix-all-samples.tsv

```diff
@@ -4256,7 +4256,6 @@ TCGA-E8-A3X7-01	0	8.05	8.19	9.49	5.06	10.9	8.27	10.8	7.01	9.11	3.23	6.07	0	5.62
 TCGA-E8-A413-01	0	8.15	8.44	9.42	6	10.6	9.58	10.3	7.56	8.67	2.42	5.49	0	6.32	13.6
 TCGA-E8-A414-01	0	7.97	8.4	9.7	5.71	10.5	9.63	10.8	7.28	8.57	2.99	5.66	0	6.97	12.8
 TCGA-E8-A415-01	0	7.94	7.22	9.44	5.22	10.4	9.28	10.6	7.2	8.99	2	5.65	0	6.91	13.2
-TCGA-E8-A416-01	0	7.81	6.63	9.39	4.26	10.5	9.77	8.77	7.11	9.2	2.25	6.27	0	7	13.2
```

Comment:
Interesting -- one sample was removed, meaning they must have had few mutations, all of which didn't map.